app/services/users/profile.rb

```diff
@@ -70,6 +70,7 @@ def profile
         loa: user.loa,
         multifactor: user.multifactor,
         verified: user.loa3?,
+        sign_in: user.identity.sign_in,
         # FIXME: this wont be necessary after FE makes appropriate changes
```

Comment:
@saneshark @narin - can this comment be removed now, or are the changes in this PR something different?